### ISSUE TITLE
Enable AsciiMath support

### ISSUE THREAD

**Issue by bhack** [tensorflow/addons#2174]
**Describe the feature and the current behavior/state.**
It would be nice to get Ascimath support in docstring to have source code level math readibility and Matjax rendering in Docs
**Relevant information**
http://docs.mathjax.org/en/latest/basic/mathematics.html#asciimath-input

**Which API type would this fall under (layer, metric, optimizer, etc.)**
Docs
**Any other info.**
I don't know if we need to enable this at TF Doc level /cc @lamberta  What do you think?

**Comment by Harsh188** [comment]
I'd like to help out with this :)

**Comment by lamberta** [comment]
Hmm. Backticks for MathJax would seem to conflict with our code/auto-link format for ref docs. Not sure if it's supported in Colab, yet. And if it requires upgrading our MathJax on the site then that's a long road. cc: @MarkDaoust

**Comment by MarkDaoust** [comment]
interesting idea, but +1 to both of @lamberta's points.

1. It would be doing a lot of guessing about what's math vs. what's code. 

2. We try to make things uniform between docstrings, MD files and IPYNB files, and we try to make sure things work on github, colab, and tensorflow.org. 

So I don't think this is really something we can/should do right now.
